README.md

```diff
@@ -296,6 +296,17 @@ db.Update(func(tx *bolt.Tx) error {
 })
 ```
 
+You can retrieve an existing bucket using the `Tx.Bucket()` function:
```

Comment:
nit: Precisely, it should a method instead of function. I see that other places also use function. It's OK to keep it as it's. Please feel free to update it in a separate PR.